tests/test_postfix.py

```diff
@@ -0,0 +1,28 @@
+"""This module contains the tests for the postfix container, the image with postfix, sendmail and mailq pre-installed."""
+
+from bci_tester.data import POSTFIX_CONTAINERS
+
+CONTAINER_IMAGES = POSTFIX_CONTAINERS
+
+
+def test_postfix_status(auto_container):
+    # check if Postfix is running inside the container.
+    assert "localhost" in auto_container.connection.check_output(
+        "echo ${SERVER_HOSTNAME}"
+    )
+
+    auto_container.connection.check_output("postfix status")
```

Comment:
```suggestion
    assert "the Postfix mail systein is running" in auto_container.connection.check_output("postfix status")
```

Reply:
updated to assert on "the Postfix mail system is running" string.

Just a note, `postfix status` redirects ^ string to stderr, so adjusted accordingly.

Thanks!